lib/components/src/controls/types.ts

```diff
@@ -80,7 +75,6 @@ export type ControlType =
   | 'text';
 
 export type Control =
-  | ArrayConfig
```

Comment:
I suspect that technically this is a breaking change. Consumers could have been passing a `separator` param which has been ignored since we switched to ObjectControl. Do we want a deprecation warning around this or just do it?

Reply:
KISS